### DISCUSSION TITLE
Status of the support for Franka Emika Robots

### DISCUSSION THREAD

**Discussion by robin-gdwl** [visose/Robots#255] [Q&A] [answered]
Hello, 
I would like to ask what the current status of support for Franka Emika Robots is. 
I have seen that there is "Preview Support" in [V1.6](https://github.com/visose/Robots/blob/3ed7b961786899fa86f8fdaaaca38e87eba7c015/RELEASE#L26). 
Is there already a Robot Library with a Franka Robot? 
What is missing to fully support Franka Robots and would pull requests in that area be accepted? 

Thank you for your time.

**Comment by visose** [comment, marked as answer]
Hi @robin-gdwl, I wrote some instructions on how to use on this thread: https://github.com/visose/Robots/discussions/253
The Bartlett library has a Franka Emika Panda robot.
I guess it's in preview because it hasn't been tested much. Sure, you can make pull requests but for new features we can discuss it beforehand to make sure it will be suitable for this project. If it's something specialized you can always fork it or create a plugin that uses Robots.

One change I'd like to make is that it currently uses a different solver on Windows and Mac, in Windows it works noticable better as it uses the Orocos KDL solver. I'd like to use a single one on both.

**Reply by robin-gdwl** [reply]
Thank You very much for the reply. 
The Bartlett Library is probably the only one I did not check, but makes ssense that it is located there.
I will test it on Windows and Mac and see what the current state is and where I might be able to contribute.
